shared/constants/time.js

```diff
@@ -0,0 +1,5 @@
+export const MILLISECOND = 1;
```

Comment:
After merge I'll put up a change to use these across the codebase where applicable